rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -87,12 +99,12 @@ class PlayerImpl
 public:
   using callback_handle_t = Player::callback_handle_t;
   using play_msg_callback_t = Player::play_msg_callback_t;
+  using ReaderStorageOptionsPair = Player::ReaderStorageOptionsPair;
```

Comment:
Please use snake style naming as above for consistency

Reply:
renamed in ca23e7fae501e6a2950f529ef0dafa2f37dd1f26